tests/e2e/script.js

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
+
+/**
+ * E2E Test Script Runner
+ *
+ * This file was originally copied from the @wordpress/scripts package, version 12.0.0:
+ * https://github.com/WordPress/gutenberg/blob/8e06f0d212f89adba9099106497117819adefc5a/packages/scripts/scripts/test-e2e.js
+ * This file is subject to the Mozilla Public License 2.0, as it is from the `@wordpress/scripts` package, released under the same license: https://github.com/WordPress/gutenberg/blob/trunk/LICENSE.md.
+ */
+
+// Do this as the first thing so that any code reading it knows the right env.
+process.env.BABEL_ENV = 'test';
+process.env.NODE_ENV = 'test';
+
+// Makes the script crash on unhandled rejections instead of silently
+// ignoring them. In the future, promise rejections that are not handled will
+// terminate the Node.js process with a non-zero exit code.
+process.on( 'unhandledRejection', ( err ) => {
+	throw err;
+} );
+
+/**
+ * External dependencies
+ */
+/* eslint-disable-next-line jest/no-jest-import */
+const jest = require( 'jest' );
+const { sync: spawn } = require( 'cross-spawn' );
+
+// getArgsFromCLI inlined from @wordpress/scripts utils/process.js v12.0.0.
+// https://github.com/WordPress/gutenberg/blob/8e06f0d212f89adba9099106497117819adefc5a/packages/scripts/utils/process.js#L1-L11
+
+const getArgsFromCLI = ( excludePrefixes ) => {
+	const args = process.argv.slice( 2 );
+	if ( excludePrefixes ) {
+		return args.filter( ( arg ) => {
+			return ! excludePrefixes.some( ( prefix ) =>
+				arg.startsWith( prefix )
+			);
+		} );
+	}
+	return args;
+};
+
+// getArgFromCLI and hasArgInCLI inlined from @wordpress/scripts utils/cli.js v12.0.0.
+// https://github.com/WordPress/gutenberg/blob/8e06f0d212f89adba9099106497117819adefc5a/packages/scripts/utils/cli.js#L13-L22
```

Comment:
Thanks for including the commit SHA and not using `trunk` so the lines don't move around 👍🏻